### PR TITLE
ci(docs): Use Vercel deploy hook for docs sync

### PR DESCRIPTION
**Pull request by BYK**
See getsentry/sentry-docs#4080 for more info.

### CODE REVIEW

File: .github/workflows/sentry-docs.yml [hunk 1]
```diff
@@ -6,24 +6,8 @@ on:
 
 jobs:
   bump:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```
Comment (untitaker) [inline .github/workflows/sentry-docs.yml:9]: do we need this line at all?

**Review by untitaker — APPROVED** [review]
I don't own this code so take my review with a grain of salt

**Review by NisanthanNanthakumar — APPROVED** [review]

File: .github/workflows/sentry-docs.yml (hunk 1 above)
Comment (BYK) [reply .github/workflows/sentry-docs.yml:9]: Probably not.

**Comment by BYK** [comment]
Closing in favor of auto-versioning per push to sentry-api-schema. This will allow us to keep the embedded SHA.

**Pull request closed without merging**